### PR TITLE
Feature/frontend/notification bell bug

### PR DESCRIPTION
**Pull request by WillyST98**
fix notification bell issue, removed navbutton from sidebar (moved to header)

### CODE REVIEW

File: Online-Messaging-App/src/app/home/header/header.component.ts [hunk 1]
```diff
@@ -106,6 +111,14 @@ export class HeaderComponent implements OnInit {
         this.open = !this.open;
     }
 
+    switchDisplay(value: string): void {
+        this.switchEvent.emit(value);
+
+        if (value === "profile") {
```
Comment (dBaiko) [inline Online-Messaging-App/src/app/home/header/header.component.ts]: use the constant you have on line 53 here

File: Online-Messaging-App/src/app/home/header/header.component.scss [hunk 2]
```diff
@@ -43,8 +43,9 @@ header {
 //}
 .floater {
     float: right;
-    width: 20%;
+    //width: 20%;
```
Comment (dBaiko) [inline Online-Messaging-App/src/app/home/header/header.component.scss]: remove this

File: Online-Messaging-App/src/app/home/header/header.component.html [hunk 3]
```diff
@@ -3,32 +3,35 @@
```
Comment (dBaiko) [inline Online-Messaging-App/src/app/home/header/header.component.html:3]: please right click on the src folder of Online-Messaging-App and click "Reformat with Prettier" before you push up again

**Review by dBaiko — CHANGES_REQUESTED** [review]

File: Online-Messaging-App/src/app/home/header/header.component.ts (hunk 1 above)
Comment (WillyST98) [reply Online-Messaging-App/src/app/home/header/header.component.ts]: done

File: Online-Messaging-App/src/app/home/header/header.component.scss (hunk 2 above)
Comment (WillyST98) [reply Online-Messaging-App/src/app/home/header/header.component.scss]: done

File: Online-Messaging-App/src/app/home/header/header.component.html (hunk 3 above)
Comment (WillyST98) [reply Online-Messaging-App/src/app/home/header/header.component.html:3]: done

**Review by dBaiko — APPROVED** [review]

**Pull request merged**